MediaPlayer/MediaControls.js

```diff
@@ -0,0 +1,150 @@
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Button from '../Button';
+import Icon from '../Icon';
+
+import $L from '../internal/$L';
+import css from './MediaControls.module.less';
```

Comment:
Please reorder the import like below.

```suggestion
import $L from '../internal/$L';
import Button from '../Button';
import Icon from '../Icon';

import css from './MediaControls.module.less';
```